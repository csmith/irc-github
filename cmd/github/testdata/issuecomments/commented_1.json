{
  "action": "created",
  "issue": {
    "url": "https://api.github.com/repos/greboid/irc/issues/6",
    "repository_url": "https://api.github.com/repos/greboid/irc",
    "labels_url": "https://api.github.com/repos/greboid/irc/issues/6/labels{/name}",
    "comments_url": "https://api.github.com/repos/greboid/irc/issues/6/comments",
    "events_url": "https://api.github.com/repos/greboid/irc/issues/6/events",
    "html_url": "https://github.com/greboid/irc/issues/6",
    "id": 584325795,
    "node_id": "MDU6SXNzdWU1ODQzMjU3OTU=",
    "number": 6,
    "title": "GitHub: Handle pull requests",
    "user": {
      "login": "csmith",
      "id": 189133,
      "node_id": "MDQ6VXNlcjE4OTEzMw==",
      "avatar_url": "https://avatars0.githubusercontent.com/u/189133?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/csmith",
      "html_url": "https://github.com/csmith",
      "followers_url": "https://api.github.com/users/csmith/followers",
      "following_url": "https://api.github.com/users/csmith/following{/other_user}",
      "gists_url": "https://api.github.com/users/csmith/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/csmith/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/csmith/subscriptions",
      "organizations_url": "https://api.github.com/users/csmith/orgs",
      "repos_url": "https://api.github.com/users/csmith/repos",
      "events_url": "https://api.github.com/users/csmith/events{/privacy}",
      "received_events_url": "https://api.github.com/users/csmith/received_events",
      "type": "User",
      "site_admin": false
    },
    "labels": [
      {
        "id": 1873189850,
        "node_id": "MDU6TGFiZWwxODczMTg5ODUw",
        "url": "https://api.github.com/repos/greboid/irc/labels/enhancement",
        "name": "enhancement",
        "color": "a2eeef",
        "default": true,
        "description": "New feature or request"
      }
    ],
    "state": "closed",
    "locked": false,
    "assignee": null,
    "assignees": [

    ],
    "milestone": null,
    "comments": 1,
    "created_at": "2020-03-19T10:54:22Z",
    "updated_at": "2020-03-19T20:51:24Z",
    "closed_at": "2020-03-19T20:51:24Z",
    "author_association": "CONTRIBUTOR",
    "body": "The GitHub plugin doesn't currently announce pull requests, it'd be nice if it did."
  },
  "comment": {
    "url": "https://api.github.com/repos/greboid/irc/issues/comments/601410689",
    "html_url": "https://github.com/greboid/irc/issues/6#issuecomment-601410689",
    "issue_url": "https://api.github.com/repos/greboid/irc/issues/6",
    "id": 601410689,
    "node_id": "MDEyOklzc3VlQ29tbWVudDYwMTQxMDY4OQ==",
    "user": {
      "login": "greboid",
      "id": 189618,
      "node_id": "MDQ6VXNlcjE4OTYxOA==",
      "avatar_url": "https://avatars0.githubusercontent.com/u/189618?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/greboid",
      "html_url": "https://github.com/greboid",
      "followers_url": "https://api.github.com/users/greboid/followers",
      "following_url": "https://api.github.com/users/greboid/following{/other_user}",
      "gists_url": "https://api.github.com/users/greboid/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/greboid/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/greboid/subscriptions",
      "organizations_url": "https://api.github.com/users/greboid/orgs",
      "repos_url": "https://api.github.com/users/greboid/repos",
      "events_url": "https://api.github.com/users/greboid/events{/privacy}",
      "received_events_url": "https://api.github.com/users/greboid/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2020-03-19T20:51:24Z",
    "updated_at": "2020-03-19T20:51:24Z",
    "author_association": "OWNER",
    "body": "I believe this should now be resolved."
  },
  "repository": {
    "id": 243344149,
    "node_id": "MDEwOlJlcG9zaXRvcnkyNDMzNDQxNDk=",
    "name": "irc",
    "full_name": "greboid/irc",
    "private": false,
    "owner": {
      "login": "greboid",
      "id": 189618,
      "node_id": "MDQ6VXNlcjE4OTYxOA==",
      "avatar_url": "https://avatars0.githubusercontent.com/u/189618?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/greboid",
      "html_url": "https://github.com/greboid",
      "followers_url": "https://api.github.com/users/greboid/followers",
      "following_url": "https://api.github.com/users/greboid/following{/other_user}",
      "gists_url": "https://api.github.com/users/greboid/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/greboid/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/greboid/subscriptions",
      "organizations_url": "https://api.github.com/users/greboid/orgs",
      "repos_url": "https://api.github.com/users/greboid/repos",
      "events_url": "https://api.github.com/users/greboid/events{/privacy}",
      "received_events_url": "https://api.github.com/users/greboid/received_events",
      "type": "User",
      "site_admin": false
    },
    "html_url": "https://github.com/greboid/irc",
    "description": "Plugin based IRC library for Go",
    "fork": false,
    "url": "https://api.github.com/repos/greboid/irc",
    "forks_url": "https://api.github.com/repos/greboid/irc/forks",
    "keys_url": "https://api.github.com/repos/greboid/irc/keys{/key_id}",
    "collaborators_url": "https://api.github.com/repos/greboid/irc/collaborators{/collaborator}",
    "teams_url": "https://api.github.com/repos/greboid/irc/teams",
    "hooks_url": "https://api.github.com/repos/greboid/irc/hooks",
    "issue_events_url": "https://api.github.com/repos/greboid/irc/issues/events{/number}",
    "events_url": "https://api.github.com/repos/greboid/irc/events",
    "assignees_url": "https://api.github.com/repos/greboid/irc/assignees{/user}",
    "branches_url": "https://api.github.com/repos/greboid/irc/branches{/branch}",
    "tags_url": "https://api.github.com/repos/greboid/irc/tags",
    "blobs_url": "https://api.github.com/repos/greboid/irc/git/blobs{/sha}",
    "git_tags_url": "https://api.github.com/repos/greboid/irc/git/tags{/sha}",
    "git_refs_url": "https://api.github.com/repos/greboid/irc/git/refs{/sha}",
    "trees_url": "https://api.github.com/repos/greboid/irc/git/trees{/sha}",
    "statuses_url": "https://api.github.com/repos/greboid/irc/statuses/{sha}",
    "languages_url": "https://api.github.com/repos/greboid/irc/languages",
    "stargazers_url": "https://api.github.com/repos/greboid/irc/stargazers",
    "contributors_url": "https://api.github.com/repos/greboid/irc/contributors",
    "subscribers_url": "https://api.github.com/repos/greboid/irc/subscribers",
    "subscription_url": "https://api.github.com/repos/greboid/irc/subscription",
    "commits_url": "https://api.github.com/repos/greboid/irc/commits{/sha}",
    "git_commits_url": "https://api.github.com/repos/greboid/irc/git/commits{/sha}",
    "comments_url": "https://api.github.com/repos/greboid/irc/comments{/number}",
    "issue_comment_url": "https://api.github.com/repos/greboid/irc/issues/comments{/number}",
    "contents_url": "https://api.github.com/repos/greboid/irc/contents/{+path}",
    "compare_url": "https://api.github.com/repos/greboid/irc/compare/{base}...{head}",
    "merges_url": "https://api.github.com/repos/greboid/irc/merges",
    "archive_url": "https://api.github.com/repos/greboid/irc/{archive_format}{/ref}",
    "downloads_url": "https://api.github.com/repos/greboid/irc/downloads",
    "issues_url": "https://api.github.com/repos/greboid/irc/issues{/number}",
    "pulls_url": "https://api.github.com/repos/greboid/irc/pulls{/number}",
    "milestones_url": "https://api.github.com/repos/greboid/irc/milestones{/number}",
    "notifications_url": "https://api.github.com/repos/greboid/irc/notifications{?since,all,participating}",
    "labels_url": "https://api.github.com/repos/greboid/irc/labels{/name}",
    "releases_url": "https://api.github.com/repos/greboid/irc/releases{/id}",
    "deployments_url": "https://api.github.com/repos/greboid/irc/deployments",
    "created_at": "2020-02-26T19:09:37Z",
    "updated_at": "2020-03-19T20:38:32Z",
    "pushed_at": "2020-03-19T20:38:30Z",
    "git_url": "git://github.com/greboid/irc.git",
    "ssh_url": "git@github.com:greboid/irc.git",
    "clone_url": "https://github.com/greboid/irc.git",
    "svn_url": "https://github.com/greboid/irc",
    "homepage": "",
    "size": 180,
    "stargazers_count": 0,
    "watchers_count": 0,
    "language": "Go",
    "has_issues": true,
    "has_projects": true,
    "has_downloads": true,
    "has_wiki": true,
    "has_pages": false,
    "forks_count": 1,
    "mirror_url": null,
    "archived": false,
    "disabled": false,
    "open_issues_count": 1,
    "license": {
      "key": "mit",
      "name": "MIT License",
      "spdx_id": "MIT",
      "url": "https://api.github.com/licenses/mit",
      "node_id": "MDc6TGljZW5zZTEz"
    },
    "forks": 1,
    "open_issues": 1,
    "watchers": 0,
    "default_branch": "master"
  },
  "sender": {
    "login": "greboid",
    "id": 189618,
    "node_id": "MDQ6VXNlcjE4OTYxOA==",
    "avatar_url": "https://avatars0.githubusercontent.com/u/189618?v=4",
    "gravatar_id": "",
    "url": "https://api.github.com/users/greboid",
    "html_url": "https://github.com/greboid",
    "followers_url": "https://api.github.com/users/greboid/followers",
    "following_url": "https://api.github.com/users/greboid/following{/other_user}",
    "gists_url": "https://api.github.com/users/greboid/gists{/gist_id}",
    "starred_url": "https://api.github.com/users/greboid/starred{/owner}{/repo}",
    "subscriptions_url": "https://api.github.com/users/greboid/subscriptions",
    "organizations_url": "https://api.github.com/users/greboid/orgs",
    "repos_url": "https://api.github.com/users/greboid/repos",
    "events_url": "https://api.github.com/users/greboid/events{/privacy}",
    "received_events_url": "https://api.github.com/users/greboid/received_events",
    "type": "User",
    "site_admin": false
  }
}