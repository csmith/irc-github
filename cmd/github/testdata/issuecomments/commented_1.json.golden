[greboid/irc] greboid commented on issue GitHub: Handle pull requests - https://github.com/greboid/irc/issues/6